src/apps/abracadabra/ethereum/abracadabra.cauldron.contract-position-fetcher.ts

```diff
@@ -54,7 +54,7 @@ export class EthereumAbracadabraCauldronContractPositionFetcher implements Posit
   constructor(
     @Inject(AbracadabraCauldronContractPositionHelper)
     private readonly abracadabraCauldronContractPositionHelper: AbracadabraCauldronContractPositionHelper,
-  ) {}
+  ) { }
```

Comment:
I think this is probably your IDE that is incorrectly formatting; rely on the `.prettierrc` for formatting at the root of Studio if possible 🙏